scripts/perf/datenlord-perf.yaml

```diff
@@ -32,14 +32,17 @@ spec:
           imagePullPolicy: "IfNotPresent"
           args:
             - "start_async_fuse"
-            - "--capacity=$(CACHE_CAPACITY)"
-            - "--etcd=$(ETCD_ENDPOINT)"
-            - "--mountpoint=$(FUSE_MOUNT_DIR)"
-            - "--volume_info=$(FUSE_VOLUME_INFO)"
-            - "--volume_type=none"
-            - "--nodeid=$(NODE_ID)"
-            - "--nodeip=$(NODE_IP)"
-            - "--serverport=8800"
+            - "--storage-cache-capacity=$(CACHE_CAPACITY)"
+            - "--kv-server-list=$(ETCD_ENDPOINT)"
+            - "--mount-path=$(FUSE_MOUNT_DIR)"
+            - "--storage-type=none"
+            - "--node-name=$(NODE_ID)"
+            - "--node-ip=$(NODE_IP)"
+            - "--server-port=8800"
+            - "--storage-s3-access-key-id test"
+            - "--storage-s3-secret-access-key test1234"
+            - "--storage-s3-endpoint http://127.0.0.1:9000"
```

Comment:
It's `storage-s3-endpoint-url` but not `storage-s3-endpoint`. `CRON-CI` failed again.

Reply:
I will fix this later.

Reply:
fixed.